segmentstore/server/host/src/test/java/io/pravega/segmentstore/server/host/ServiceStarterTest.java

```diff
@@ -80,6 +85,18 @@ public void testCuratorClientCreation() throws Exception {
         Assert.assertEquals("HealthContributor should report an 'DOWN' Status.", Status.DOWN, zkStatus);
     }
 
+    /**
+     * Check health of SegmentContainerRegistry
+     */
+    @Test
+    public void testSegmentContainerRegistryHealth() {
+        @Cleanup
+        SegmentContainerRegistryHealthContributor segmentContainerRegistryHealthContributor = new SegmentContainerRegistryHealthContributor(serviceStarter.getServiceBuilder().getSegmentContainerRegistry());
+        Health.HealthBuilder builder = Health.builder().name(segmentContainerRegistryHealthContributor.getName());
+        Status status = segmentContainerRegistryHealthContributor.doHealthCheck(builder);
+        Assert.assertEquals("HealthContributor should report an 'UP' Status.", Status.UP, status);
```

Comment:
Can you test also the transition to "DOWN" state?

Reply:
It cant be tested here. It is being tested here instead: `SegmentContainerRegistryHealthContributorTest`